js/app.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener(`DOMContentLoaded`, () =>{
+
+	const form = document.querySelector(`#registrar`);
+	const formInput = form.querySelector(`input`);
+	const invitedUl = document.querySelector(`#invitedList`);
+	const hideNotResponded = document.querySelector(`.main input`);
+
+	function newElement(elementParent, element) {
+		const name = document.createElement(element);
+		elementParent.appendChild(name)
+		return name
+	}
+
+	function text(elementParent, element, text) {
```

Comment:
Ta nazwa funkcji niewiele mówi i jest myląca (jest też spora szansa, że trafisz w zewn. bibliotece na metodę o podobnej nazwie i kod będzie sypał Ci błędem). Jeśli chcesz wyrzucić tę funkcjonalność do funkcji to coś w stylu `instertTextIntoElement` byłoby lepsze. 

Dodatkowo Twoje funkcje powinny mieć single area of responsibility. W tym wypadku masz funkcję `text`, która tworzy element i ładuje do niego text - o jedną funkcjonalność za dużo ;)